<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Неокогнитрон v0.9beta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="parametersTab">
       <attribute name="title">
        <string>Параметры</string>
       </attribute>
       <layout class="QGridLayout" name="parametersLayout">
        <item row="0" column="0">
         <widget class="Parameters" name="parametersWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="patternsTab">
       <attribute name="title">
        <string>Выборки</string>
       </attribute>
       <layout class="QGridLayout" name="patternsLayout">
        <item row="0" column="0">
         <widget class="PatternsWidget" name="patternsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="weightsTab">
       <attribute name="title">
        <string>Веса</string>
       </attribute>
       <layout class="QGridLayout" name="weightsLayout">
        <item row="0" column="0">
         <widget class="TrainingResultsWidget" name="weightsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="interactiveTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Интерактивная проверка</string>
       </attribute>
       <layout class="QGridLayout" name="interactiveLayout">
        <item row="0" column="0">
         <widget class="TestingWidget" name="interactiveWidget" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="batchTab">
       <attribute name="title">
        <string>Пакетная проверка</string>
       </attribute>
       <layout class="QGridLayout" name="batchLayout">
        <item row="0" column="0">
         <widget class="MultitestingWidget" name="batchWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Создать и обучить</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Отладочный вывод</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PatternsWidget</class>
   <extends>QWidget</extends>
   <header>patternswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestingWidget</class>
   <extends>QWidget</extends>
   <header>testingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MultitestingWidget</class>
   <extends>QWidget</extends>
   <header>multitestingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingResultsWidget</class>
   <extends>QWidget</extends>
   <header>trainingresultswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Parameters</class>
   <extends>QWidget</extends>
   <header>parameters.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>ActionDone(QAction*)</slot>
  <slot>UGSwitched(bool)</slot>
  <slot>AgChanged(double)</slot>
  <slot>AgoChanged(double)</slot>
  <slot>UGDecayChanged(double)</slot>
  <slot>NgChanged(int)</slot>
  <slot>LChanged(int)</slot>
  <slot>layerSelected(int)</slot>
  <slot>KsChanged(int)</slot>
  <slot>KcChanged(int)</slot>
  <slot>KcMAXChanged(int)</slot>
  <slot>NsChanged(int)</slot>
  <slot>NcChanged(int)</slot>
  <slot>mcChanged(int)</slot>
  <slot>AsChanged(double)</slot>
  <slot>AcChanged(double)</slot>
  <slot>coSChanged(double)</slot>
  <slot>coCChanged(double)</slot>
  <slot>USDecayChanged(double)</slot>
  <slot>UCExcDecayChanged(double)</slot>
  <slot>UCInhDecayChanged(double)</slot>
  <slot>UCInhDecayoChanged(double)</slot>
  <slot>thetaChanged(double)</slot>
  <slot>AcoChanged(double)</slot>
 </slots>
</ui>
